Policies/spending_guidelines.md

```diff
@@ -0,0 +1,82 @@
+# OPI Lab Budget Guidelines
+
+This document provides guidance for using OPI funds to acquire lab
+equipment.  We describe both "appropriate" and "inappropriate" use of
+equipment funds and include examples of each.  Generally, commonly
+used lab equipment generally available in the market are appropriate.
+Pre-production equipment or equipment that does not service the
+goals of the OPI lab are inappropriate.
+
+## Appropriate
+
+### General purpose servers
+
+- 1U, 2U, 3U, 4U rack servers with PCIE 3.0/4.0/5.0 slots
+- Pedestal servers
+
+### Server peripherals
+
+- Hard drives
+- RAM
+- NIC cards
+
+### Test equipment
+
+- Traffic generators
+
+### Cables
+
+- Optical cables
+- Transceivers
+- DAC cables
+
+### Storage servers
+
+### Switches
+
+- Top of Rack Layer 2/3 switches
+- Layer 1 switches
+- Programmable switches
+
+### PDUs
+
+- PDUs
+- C13/C14/C15 power cables
+
+### KVMs
+
+- KVMs
```

Comment:
not sure is needed... all modern servers have built-in KVM... maybe for DPUs... for debug...